File: ProjetoPython/Jogador.py
from Pessoa import Pessoa

class Jogador(Pessoa):

    def __init__(self, nome, time, posicao, camisa, salario):
        super().__init__(nome, time)
        self.__posicao = posicao
        self.__camisa = camisa
        self.__salario = salario
        print("Jogador Criado")

    def get_posicao (self):
        return self.__posicao

    def set_posicao (self, posicao):
        self.__posicao = posicao

    def get_camisa (self):
        return self.__camisa

    def set_camisa (self, camisa):
        self.__camisa = camisa

    def get_salario (self):
        return self.__salario

    def set_salario (self, salario):
        self.__salario = salario

    def calcularSalarioAnual (self):
        return self.__salario*12.00

    def apresentacao(self):
        print(self._nome, "é o jogador camisa", self.__camisa, "do time", self._time)

File: ProjetoJava/src/main/java/source/projetojava/Jogador.java
package source.projetojava;

/**
 *
 * @author alejb
 */
public class Jogador extends Pessoa {
  
  String posicao;
  float salario;
  int camisa;
    
  public Jogador() {

    System.out.println("Jogador criado");
        
  }
    
  public Jogador(String nome, String time, String posicao, int camisa, float salario) {
  
    super (nome, time);
    this.posicao = posicao;
    this.camisa = camisa;
    this.salario = salario;
    System.out.println("Jogador criado");
        
  }

  @Override
  public void apresentacao () {
      
    System.out.println(this.nome + " é o jogador camisa " + this.camisa + " do time " + this.time + ".");
    
  }

  public void setPosicao(String posicao) {
    this.posicao = posicao;
  }

  public String getPosicao() {
    return this.posicao;
  }

  public void setCamisa(int camisa) {
    this.camisa = camisa;
  }

  public int getCamisa() {
    return this.camisa;
  }

  public void setSalario(float salario) {
    this.salario = salario;
  }
  
  public float getSalario() {
    return this.salario;
  }
  
  public float calcularSalarioAnual () {  
    return (float) (this.salario*12.00);
  }
    
}
